apps/pom.xml

```diff
@@ -1025,9 +1025,14 @@
                     <failOnWarnings>false</failOnWarnings>
                     <detectJavaApiLink>false</detectJavaApiLink>
                     <!-- Exclude javadocs on certain generated files, reduces build warnings -->
-                    <excludePackageNames>gov.cms.mpsm.rda.*</excludePackageNames>
-                    <excludePackageNames>gov.cms.bfd.model.rif.entities.*</excludePackageNames>
-                    <excludePackageNames>gov.cms.bfd.model.rda.entities.*</excludePackageNames>
+                    <excludePackageNames>
+                        gov.cms.bfd.pipeline.rda.grpc.*;
+                        gov.cms.mpsm.rda.v1;
+                        gov.cms.mpsm.rda.v1.*;
```

Comment:
Wouldnt the first line here be equivalent to the second? (I'd assume covering gov.cms.mpsm.rda.v1 would also exclude children along the path)

Reply:
Resolved